__tests__/app.test.js

```diff
@@ -86,3 +86,29 @@ describe("GET", () => {
       });
   });
 });
+
+describe("GET", () => {
+  it("should respond with an array of review objects with expected properties", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        const bodyReviews = body.reviews;
+        expect(bodyReviews).toHaveLength(13);
+        expect(bodyReviews).toBeInstanceOf(Array);
+        bodyReviews.forEach((review) => {
+          expect(review).toHaveProperty("review_id", expect.any(Number));
+          expect(review).toHaveProperty("title", expect.any(String));
+          expect(review).toHaveProperty("designer", expect.any(String));
+          expect(review).toHaveProperty("review_img_url", expect.any(String));
+          expect(review).toHaveProperty("votes", expect.any(Number));
+          expect(review).toHaveProperty("category", expect.any(String));
+          expect(review).toHaveProperty("owner", expect.any(String));
+          expect(review).toHaveProperty("comment_count", expect.any(String));
+          expect(review).toHaveProperty("created_at", expect.any(String));
+          expect(Object.keys(review)).toHaveLength(9);
+          expect(bodyReviews).toBeSortedBy("created_at", { descending: true });
```

Comment:
Great use of jest-sorted. I would have this in its own test block just to separate 'returns all review objects' from 'returns review objects in reverse chronological order'